gpu_docker/docker-build-12.sh

```diff
@@ -0,0 +1,13 @@
+COMMIT=${1}
+COMMIT="$(echo "${COMMIT}" | tr -d '[:space:]')"
+export COMMIT
+
+NVIDIAVERSION="12.0.1-devel-ubuntu20.04"
+export NVIDIAVERSION
+NO_TAG="ghcr.io/${2}/pyafq_gpu_cuda_12"
```

Comment:
Couldn't `pyafq_gpu_cuda_12` be an argument to the script, instead of duplicating it?